src/scripts/content/asana.js

```diff
@@ -61,7 +61,7 @@ togglbutton.render('.BoardCard.BoardColumnCardsContainer-item:not(.toggl)', {obs
   var link,
     container = $('.BoardCardMetadata-left', elem),
     description = $('.BoardCard-name', elem).textContent,
-    project = $('#project_pane_header_name').textContent;
+    project = $('.DeprecatedNavigationLink.SidebarItemRow.is-selected').textContent;
```

Comment:
I think we maybe should drop the `DeprecatedNavigationLink` class. `SidebarItemRow` seems enough (tried the querySelector... both returned the same number.
I say this because `DeprecatedNavigationLink` seems like an obvious candidate to be removed soon. Probably will save us some time in the future.